### DISCUSSION TITLE
FatalError: `Process abort signal` is detected by the operating system.

### DISCUSSION THREAD

**Discussion by SolarKnight1** [PaddlePaddle/PaddleOCR#12726] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem
使用了python PPOCRLabel.py --lang ch这个命令后出现这个错误，我看了挺多的issue，都试了试，但还是这个问题

- 系统环境/System Environment：
- 版本号/Version：Paddle： 最新版跟2.3.2，2.4.2都试了 PaddleOCR：最新版，2.5.0也试了 问题相关组件/Related components： python 3.7.0
- 运行指令/Command Code：python PPOCRLabel.py --lang ch
- 完整报错/Complete Error Message：
/root/anaconda3/lib/python3.7/site-packages/dask/compatibility.py:22: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  PY_VERSION = LooseVersion(".".join(map(str, sys.version_info[:3])))
/root/anaconda3/lib/python3.7/site-packages/skimage/morphology/_skeletonize.py:241: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  0, 1, 1, 0, 0, 1, 0, 0, 0], dtype=np.bool)
/root/anaconda3/lib/python3.7/site-packages/skimage/morphology/_skeletonize.py:256: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  0, 0, 0, 0, 0, 0, 0, 0, 0], dtype=np.bool)
/root/anaconda3/lib/python3.7/site-packages/skimage/segmentation/random_walker_segmentation.py:50: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  if Version(scipy.__version__) >= Version('1.1'):
Failed to create wl_display (No such file or directory)
qt.qpa.plugin: Could not load the Qt platform plugin "wayland" in "/root/anaconda3/lib/python3.7/site-packages/cv2/qt/plugins" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb, eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, webgl.

--------------------------------------
C++ Traceback (most recent call last):
--------------------------------------
No stack trace in paddle, may be caused by external reasons.

----------------------
Error Message Summary:
----------------------
FatalError: `Process abort signal` is detected by the operating system.
  [TimeInfo: *** Aborted at 1693893135 (unix time) try "date -d @1693893135" if you are using GNU date ***]
  [SignalInfo: *** SIGABRT (@0x4245) received by PID 16965 (TID 0x7f9a00802740) from PID 16965 ***]

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by SolarKnight1** [comment]
而且我在windows也能正常启动，但是太慢了，还容易卡死，但换到服务器上就不行

**Comment by ghsy158** [comment]
报同样的错，这个问题到底是什么原因呢，麻烦paddle的同事帮忙解答一下
